pkg/controller/provider/container/vsphere/collector.go

```diff
@@ -401,16 +400,7 @@ next:
 		}
 		updateSet := response.Returnval
 		if updateSet == nil {
-			err := r.connect(ctx)
-			if err != nil {
-				r.log.Error(
-					err,
-					"failed to connect.",
-					"url",
-					r.url)
-				time.Sleep(RetryDelay)
-			}
-			continue next
+			continue
```

Comment:
This will _continue_ to be _for {_ which is equivalent to using the label.
label not needed.